frappe/types/exporter.py

```diff
@@ -22,16 +22,22 @@
 end_block = "# end: auto-generated types"
 
 type_code_block_template = """{start_block}
-
 # This code is auto-generate. Do not modify anything in this block.
-from frappe.types import DF
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+	from frappe.types import DF
 
 {child_table_imports}
 
 {fields}
+
 {end_block}
 """
 
+INDENT = "\t"
```

Comment:
Can this lead to syntax (mixed indentation) errors when modules use spaces instead?

Reply:
Yeah, will guess from file instead.